homeassistant/components/mutesync/__init__.py

```diff
@@ -15,6 +15,9 @@
 
 PLATFORMS = ["binary_sensor"]
 
+UPDATE_INTERVAL_NOT_IN_MEETING = timedelta(seconds=10)
+UPDATE_INTERVAL_IN_MEETING = timedelta(seconds=1)
```

Comment:
Normally we don't allow lower update interval than 5 seconds for polling integrations.